intrastat_product/views/account_config_settings.xml

```diff
@@ -14,6 +14,7 @@
           <field name="intrastat_transaction_out_refund"/>
           <field name="intrastat_transaction_in_invoice"/>
           <field name="intrastat_transaction_in_refund"/>
+          <field name="country_id" invisible="1"/>
```

Comment:
I have a doubt.

This field needs be added in order to fix a domain error for `intrastat_region_id` field.
But I don't understand what is the reason to adds a field with domain but invisible in the case of `intrastat_region_id`

What about remove from the view `intrastat_region_id` since that is not used from other domain or another field?
And both spent time to compute them (Because they are not using `store=True`)